packages/db/src/meta/pouch/adapters/base.ts

```diff
@@ -18,6 +18,11 @@ import type { Workspace, Historical } from "@truffle/db/meta/data";
 
 import type { Definition, Definitions } from "@truffle/db/meta/pouch/types";
 
+export interface DatabasesOptions<C extends Collections> {
+  definitions: Definitions<C>;
+  settings: any;
```

Comment:
Does this have to be an `any` type?

Reply:
It could be `{}` if you prefer, but it's `any` right now because it allows subclasses to define whatever settings they want.

Reply:
That makes sense! A comment to let subclass authors know the intent is better.